src/sys/windows.rs

```diff
@@ -1,6 +1,17 @@
-use winapi::{WSABUF, DWORD};
 use std::{mem, slice, u32};
 
+// declare the types we need directly here to avoid bringing
+// in the old and slow winapi 0.2 dependency.
+
+type DWORD = u32;
+type ULONG = u32;
+type CHAR = i8;
+
+struct WSABUF {
```

Comment:
This should be `#[repr(C)]` here.

Reply:
you're right, missed that, damn. will do PR